src/main/java/com/sapozhnykov/services/cart/impl/CartServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package com.sapozhnykov.services.cart.impl;
+
+import com.sapozhnykov.dao.cart.CartDao;
+import com.sapozhnykov.dao.cart.impl.CartDaoImpl;
+import com.sapozhnykov.domain.Product;
+import com.sapozhnykov.services.cart.CartService;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+
+public class CartServiceImpl implements CartService {
+    private CartDao cartDao = new CartDaoImpl();
+
+    @Override
+    public boolean addToCart(Product product) {
+        return cartDao.add(product);
```

Comment:
where is you logic?

Reply:
At the last lesson, Artem said that we would not have a Cart. I removed it from the project.